src/devenv.nix

```diff
@@ -77,12 +77,32 @@ pkgs.writeScriptBin "devenv" ''
       if [ "$(cat $procfile)" = "" ]; then
         echo "No 'processes' option defined."  
         exit 1
+      fi;
+
+      honcho_command="${pkgs.honcho}/bin/honcho start -f $procfile --env $procfileenv"
+
+      if [[ $# -eq 1 ]] && [[ $1 == "-d" ]]; then
+        echo "Starting background processes ..." 1>&2
+        echo "" 1>&2
+        nohup $honcho_command &> "$DEVENV_DIR/honcho.log" &
+        echo "$!" > "$DEVENV_DIR/honcho.pid"
       else
         echo "Starting processes ..." 1>&2
         echo "" 1>&2
-        ${pkgs.honcho}/bin/honcho start -f $procfile --env $procfileenv
+        $honcho_command
       fi
-      ;;
+    ;;
+    down)
+      if [ -r "$DEVENV_DIR/honcho.pid" ]; then
+        echo "Stopping processes ..." 1>&2
+        echo "" 1>&2
+        kill $(cat "$DEVENV_DIR/honcho.pid")
+        rm "$DEVENV_DIR/honcho.pid"
+      fi;
```

Comment:
Maybe info that no background process could be found?

Reply:
That's a good point, I will add it.